GameDefinition/Content/CaveKind.hs

```diff
@@ -140,7 +142,7 @@ casino = arena
   , cfenceTileE   = "habitat containment wall"
   , cfenceTileS   = "habitat containment wall"
   , cfenceTileW   = "habitat containment wall"
-  , cdesc         = ""
+  , cdesc         = "No longer filled with hollow-eyed gamblers, more dangerous things now lurk in the dark."
```

Comment:
Educate me: I'd put a semicolon there instead of a comma. Why is coma preferable?